### DISCUSSION TITLE
Lock volume

### DISCUSSION THREAD

**Discussion by cableghost** [File-New-Project/EarTrumpet#858] [Ideas]
I would like the ability to lock the volume of a particular app. Example would be when playing a video/movie, they tend to need increased volume, but browser doesn't. Problem is... when the browser is initially adjusted, it will be lowered when lowering overall vol after video. With the ability to lock vol, I could lock vol of browser and make overall vol adjustments at any time w/o affecting browser vol.

**Comment by riverar** [comment]
Not sure I understand. EarTrumpet today lets you individually control the volume of each app. You can adjust the overall device volume without affecting the individual apps. Are you not seeing that behavior? Can you provide a bit more detail/step by step? Perhaps are you using media hotkeys or?

**Reply by cableghost** [reply]
Thanks @riverar. I uninstalled, re-installed and that functionality seems to be working right again. Before doing so, when I adjusted the overall volume, the vol of the other apps, which I had adjusted already, were also being controlled.

On the subject of apps... Is there a way to differentiate PWS apps? This too may have contributed to what was happening.
